NEWS.md

```diff
@@ -29,3 +29,13 @@ to our output diagnostics using the ClimaDiagnostics interface.
 
 This PR also removes the AMIP paperplots function, but this
 functionality is replaced by the generalized `make_plots` function.
+
+#### Remove PostProcessor module - PR [#1022](https://github.com/CliMA/ClimaCoupler.jl/pull/1022)
+After switching to use ClimaDiagnostics.jl and ClimaAnalysis.jl
+for our diagnostics, the PostProcessor module is not needed.
+This PR follows directly from the Diagnostics module removal.
+
+### Maintenance
+- Update to JuliaFormatter v2. PR [#1024](https://github.com/CliMA/ClimaCoupler.jl/pull/1024)
```

Comment:
The release for JuliaFormatter v2 was yanked because it broke some existing codes